goon/browserassets/js/browserOutput.js

```diff
@@ -122,11 +133,11 @@ function highlightTerms(el) {
 					var newWord = null;
 					for (var i = 0; i < opts.highlightTerms.length; i++) { //Each highlight term
 						if (opts.highlightTerms[i] && words[w].toLowerCase().indexOf(opts.highlightTerms[i].toLowerCase()) > -1) { //If a match is found
-							newWord = words[w].replace(new RegExp(opts.highlightTerms[i], 'gi'), addHighlightMarkup);
+							newWord = words[w].replace("<", "&lt;").replace(new RegExp(opts.highlightTerms[i], 'gi'), addHighlightMarkup);
 							break;
 						}
 					}
-					newText += (newWord ? newWord : words[w]).replace("<", "&lt;");
+					newText += newWord || words[w];
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

> Returns expr1 if it can be converted to true; otherwise, returns expr2. Thus, when used with Boolean values, || returns true if either operand can be converted to true; if both can be converted to false, returns false.